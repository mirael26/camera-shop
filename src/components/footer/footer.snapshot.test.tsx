import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import Footer from './footer';

test('Footer renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Footer/>);
  expect(asFragment()).toMatchSnapshot();
});
