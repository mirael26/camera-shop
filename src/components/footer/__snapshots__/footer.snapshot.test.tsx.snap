// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
    data-testid="footer"
  >
    <div
      class="container"
    >
      <div
        class="footer__info"
      >
        <a
          aria-label="Переход на главную"
          class="footer__logo"
          href="/"
        >
          <svg
            aria-hidden="true"
            height="36"
            width="100"
          >
            <use
              xlink:href="#icon-logo-mono"
            />
          </svg>
        </a>
        <p
          class="footer__description"
        >
          Интернет-магазин фото- и видеотехники
        </p>
        <ul
          class="social"
        >
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу вконтакте"
              class="link"
              href="https://example.com"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-vk"
                />
              </svg>
            </a>
          </li>
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу pinterest"
              class="link"
              href="https://example.com"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-pinterest"
                />
              </svg>
            </a>
          </li>
          <li
            class="social__item"
          >
            <a
              aria-label="Переход на страницу reddit"
              class="link"
              href="https://example.com"
            >
              <svg
                aria-hidden="true"
                height="20"
                width="20"
              >
                <use
                  xlink:href="#icon-reddit"
                />
              </svg>
            </a>
          </li>
        </ul>
      </div>
      <ul
        class="footer__nav"
      >
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Навигация
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="/catalog"
              >
                Каталог
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Гарантии
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Доставка
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                О компании
              </a>
            </li>
          </ul>
        </li>
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Ресурсы
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Курсы операторов
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Блог
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Сообщество
              </a>
            </li>
          </ul>
        </li>
        <li
          class="footer__nav-item"
        >
          <p
            class="footer__title"
          >
            Поддержка
          </p>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                FAQ
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="link"
                href="https://example.com"
              >
                Задать вопрос
              </a>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </footer>
</DocumentFragment>
`;
